src/Workspaces/Core/Portable/Remote/RemoteServiceName.cs

```diff
@@ -22,9 +22,9 @@ namespace Microsoft.CodeAnalysis.Remote
         internal const string Prefix = "roslyn";
         internal const string IntelliCodeServiceName = "pythia";
         internal const string RazorServiceName = "razorLanguageService";
-        internal const string UnitTestingAnalysisServiceName = "remoteAnalysisService";
-        internal const string LiveUnitTestingServiceName = "remoteBuildService";
-        internal const string TestSourceLocationLookupServiceName = "remoteTestSourceLocationLookupService";
+        internal const string UnitTestingAnalysisServiceName = "unitTestAnalysis";
```

Comment:
@shyamnamboodiripad for checking do these names make sense

Reply:
Can you make these start with capital letter? (e.g. `UnitTestAnalysis`)

Reply:
I updated https://github.com/dotnet/roslyn/issues/42582 - previously I wrote "CamelCasing" but I actually meant "PascalCasing". Sorry about the confusion.

Reply:
Other non-Roslyn VS services use PascalCasing, so to be consistent.